package.json

```diff
@@ -18,7 +18,7 @@
 		"@guardian/atom-renderer": "1.2.2",
 		"@guardian/automat-modules": "^0.3.8",
 		"@guardian/braze-components": "^8.1.3",
-		"@guardian/commercial-core": "^5.1.1",
```

Comment:
Did you intend to remove the caret? I don't think this is a bad idea but wondered if it was intention?

Reply:
It wasn't my intention, I ran `yarn upgrade @guardian/commercial-core@5.1.3` and forgot to include the caret.

I'll put it back I think, thanks!